### DISCUSSION TITLE
connector.disconnect is not a function

### DISCUSSION THREAD

**Discussion by pemaju** [wevm/wagmi#3834] [Q&A]
while disconnect the  wallet in wagmi by wallet connect provider, this error had been threw. The error has been below mentioned.
Kindly Assist.

Error:
**connector.disconnect is not a function**

**Comment by BainWang0** [comment]
> while disconnect the wallet in wagmi by wallet connect provider, this error had been threw. The error has been below mentioned. Kindly Assist.
> 
> Error: **connector.disconnect is not a function**

Is your problem solved? I also encountered the same problem and couldn't find the reason. Please help.
<img width="1065" alt="image" src="https://github.com/wevm/wagmi/assets/167943073/2b5c7cca-e636-4235-9c28-1f0f181c6bc0">

**Comment by playground** [comment]
I'm seeing the same issue on Chrome desktop

**Comment by lindsaytyy** [comment]
have u solved that? i encountered this same issue too....got no idea at all

**Comment by dstojanovski** [comment]
Any updates on that?

The same issue appears on wagmi_flutter_web.

**Comment by cp-weng-tkspring** [comment]
same here

**Reply by jxom** [reply]
[Here](https://github.com/wevm/wagmi/issues/new?template=bug_report.yml&title=connector.disconnect%20is%20not%20a%20function) is the solution.

**Comment by mrwillis** [comment]
Issue still present at 3.4.1 and likely the same underlying cause as 

https://github.com/wevm/wagmi/discussions/4216
